app/lib/theme/ReadOnlyTheme.tsx

```diff
@@ -21,6 +21,7 @@ const readOnlyTheme: ThemeProps = {
     PhoneNumberWidget: ReadOnlyWidget,
     SelectRfpWidget: ReadOnlyWidget,
     SelectProjectStatusWidget: ReadOnlyWidget,
+    ConditionalAmountWidget: ReadOnlyWidget,
```

Comment:
Is this needed, or can we figure out a way to have rjsf default to the `ReadOnlyWidget`?

Reply:
I'm not sure, since we define these widgets to be used in our schemas, rjsf complains if they're not defined somewhere in the theme. We may be able to set a default? Maybe @mikevespi has more context on why this is needed or if there is another option. Could be an "explore" card.

Reply:
Normally this would happen by passing `disabled` to the form and it would propagate to every widget. Because we use a separate theme as opposed to passing disabled down, we (as far as I can think right now) need to specify the widget we want to render. I'll take a look into it though, there could be a clever way around the exhaustive list.